extensions/amp-ad/0.1/amp-ad-xorigin-iframe-handler.js

```diff
@@ -286,6 +324,15 @@ export class AmpAdXOriginIframeHandler {
       this.embedStateApi_.destroy();
       this.embedStateApi_ = null;
     }
+    if (this.positionObserver_) {
+      if (this.iframe) {
+        this.positionObserver_.unobserve(this.iframe);
```

Comment:
@aghassemi  Added this as positionObserver should unobserve and stop calling `getBoundingClientRect` after an element is cleaned up.